src/Inline_Script.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * Detects if an upgrade is ran.
+ *
+ * @package Yoast\Test_Helper
+ */
+
+namespace Yoast\Test_Helper;
+
+/**
+ * Class to add an inline script after a wordpress-seo script.
+ */
+class Inline_Script implements Integration {
+	/**
+	 * Holds our option instance.
+	 *
+	 * @var Option
+	 */
+	private $option;
+
+	/**
+	 * Class constructor.
+	 *
+	 * @param Option $option Our option array.
+	 */
+	public function __construct( Option $option ) {
+		$this->option = $option;
+	}
+
+	/**
+	 * Registers WordPress hooks and filters.
+	 *
+	 * @return void
+	 */
+	public function add_hooks() {
+		if ( $this->option->get( 'add_inline_script' ) === true ) {
+			add_action( 'admin_enqueue_scripts', array( $this, 'add_inline_script' ) );
+		}
+
+		add_action( 'admin_post_yoast_seo_test_inline_script', array( $this, 'handle_submit' ) );
+	}
+
+	/**
+	 * Add an inline script after the specified script.
+	 */
+	public function add_inline_script() {
+		wp_add_inline_script(
+			'yoast-seo-' . $this->option->get( 'inline_script_handle' ),
+			$this->option->get( 'inline_script' )
+		);
+	}
+
+	/**
+	 * Retrieves the controls.
+	 *
+	 * @return string The HTML to use to render the controls.
+	 */
+	public function get_controls() {
```

Comment:
This can use a bit more explanation on what it actually does.
Applying the `select` change, as described below, will help in this regard.

Also explaining what is expected in the `script` field, that it should not contain the `<script>` tags itself, will help on how to use this piece of functionality.